ballerina/types.bal

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2025 WSO2 LLC (http://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+# Configuration for the Milvus vector store.
+#
+# + collectionName - The name of the collection to use for the vector store.
+# + chunkFieldName - The name of the field to contain the chunk details
+# + topK - The number of top similar vectors to return in queries
+public type Configuration record {|
+    string collectionName;

```

Comment:
shall we provide a default collection name?

Reply:
There is no specific collection name for milvus. Shall I go on with `default` here?